### DISCUSSION TITLE
Search in Views and Folders does not find archived notes with active toggle "Show archived"

### DISCUSSION THREAD

**Discussion by xthrk** [standardnotes/forum#2887] [General] [answered]
**Describe the bug**
It is possible to display the archived notes in the views using the "Show archived" option. But when I then use the search, these archived notes are not found, although the option "Show archived" is activated.

**To Reproduce**
1. Go to Notes View
2. Click on Button for Preferences in the top
3. Activate the "Show archived" toggle
4. Click in Search-Field
5. Search for archived notes

**Comment by effieeee** [comment]
Hi @crophit, I'm afraid I couldn't reproduce this so far on desktop, web and mobile. Are you encountering this consistently on all platforms?

**Comment by xthrk** [comment, marked as answer]
Hey @effieeee, thanks for your reply. I just created some screenshots and then I found the solution:

I first activated "Show archived" in the Notes-View:
![grafik](https://user-images.githubusercontent.com/116081825/199084642-7a7effcc-ed68-40cf-89e7-72f486eab26e.png)

Now I can see non-archived and archived notes:
![grafik](https://user-images.githubusercontent.com/116081825/199084740-4fd7ee4e-1dbe-4e00-9f94-f86c197f51fa.png)

In the next step, I did a search and the archived notes were not found:
![grafik](https://user-images.githubusercontent.com/116081825/199084931-e3618bfb-639a-4dca-94ed-98d0a35812d5.png)

But then I realized the not activated "Archived"-Button below the Search-Field:
![grafik](https://user-images.githubusercontent.com/116081825/199085086-0fa280b2-ec65-43a8-9c7c-cf1d268ad34d.png)

What I learned: There are different settings for archived notes to display and search in Notes View:
* In order for archived notes to be displayed, the "Show archived" option must be enabled.
* In order for archived notes to be searched, the "Archived" button under the search box must be enabled.

Both can and must be activated separately from each other.

**Comment by xthrk** [comment]
While the "Show archived" option remains enabled for the archived notes to be displayed in the list, the "Archived" button under the search box has to be re-enabled again after restarting the desktop app or reloading the website.

**Comment by effieeee** [comment]
Ah, yes, I apologize for not considering that toggle right away 🙏 Though this does give the idea that perhaps we should consider automatically showing archived notes whenever that filter is selected on the search bar. Will keep this in mind! 🙏
